application/src/main/java/org/togetherjava/tjbot/commands/tags/TagCommand.java

```diff
@@ -20,6 +21,7 @@
     private final TagSystem tagSystem;
 
     private static final String ID_OPTION = "id";
+    private static final String MENTION_USER = "mention";
```

Comment:
Naming issue on the option. Should be called `..._OPTION` to clarify that its about a command option.

To streamline it and to stay consistent with the original UX proposal in the GH issue, maybe rename to `REPLY_TO_OPTION = "reply-to"`

Reply:
I named it mention because I am mentioning someone not exactly replying?